src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -136,9 +136,15 @@ public class ClassActions
             {
                 var allMembers = node.Members.ToList();
                 var allMethods = allMembers.OfType<MethodDeclarationSyntax>();
-                var removeMethod = allMethods.Where(m => m.Identifier.ToString() == methodName).FirstOrDefault();
-                allMembers.Remove(removeMethod);
-                node = node.RemoveNode(removeMethod, SyntaxRemoveOptions.KeepNoTrivia).NormalizeWhitespace();
+                if(allMethods.Count() > 0)
+                {
+                    var removeMethod = allMethods.Where(m => m.Identifier.ToString() == methodName);
+                    if(removeMethod.Count() > 0)
+                    {
+                        node = node.RemoveNode(removeMethod.FirstOrDefault(), SyntaxRemoveOptions.KeepNoTrivia).NormalizeWhitespace();
+                    }
```

Comment:
nit: When getting removeMethod, use FirstOrDefault, then check if it's null. This way you won't have to call FirstOrDefault again inside the condition.

Reply:
Re-tested as mentioned and updated.